jetbrains-core/src/software/aws/toolkits/jetbrains/services/codewhisperer/util/CodeWhispererFileCrawler.kt

```diff
@@ -62,6 +66,22 @@ abstract class CodeWhispererFileCrawler : FileCrawler {
         }
     }.orEmpty()
 
+    override fun listRelevantFilesInEditors(psiFile: PsiFile): List<VirtualFile> {
```

Comment:
Does this need a certain thread to execute in?

Reply:
good catch, let me wrap L72 in readAction, should be safer